debezium-core/src/main/java/io/debezium/config/Configuration.java

```diff
@@ -60,7 +60,7 @@
 @Immutable
 public interface Configuration {
 
-    public static final Pattern PASSWORD_PATTERN = Pattern.compile(".*password$|.*sasl\\.jaas\\.config$", Pattern.CASE_INSENSITIVE);
+    Pattern PASSWORD_PATTERN = Pattern.compile(".*password$|.*sasl\\.jaas\\.config$", Pattern.CASE_INSENSITIVE);
```

Comment:
Why is this not static final any longer? It really should be.

Reply:
It's an interface. everything is public static final anyway.

Reply:
@rk3rn3r  Please remove this change as in other intefaces we follow the same convention.

Reply:
@jpechane what do you mean? this code doesn't change anything? every member var already is `public static final`

Reply:
@rk3rn3r ok, my proposal then is to keep the line as is and create a task with proposal of removing those modifiers globally. If agreed upon we can add formatter rule to enforce it.

Reply:
I filed https://issues.redhat.com/browse/DBZ-2439 for this :heart: